analyses/CLK1-splicing-impact-morpholino/11-compute-binding-affinities.R

```diff
@@ -0,0 +1,76 @@
+# 11-compute-binding-affinities.R
```

Comment:
>Error in .load_package_gracefully("pwalign", "starting with BioC 3.19, ",  : 
  Could not load package pwalign. Is it installed?

  Note that starting with BioC 3.19, calling pairwiseAlignment() requires
  the pwalign package. Please install it with:

    BiocManager::install("pwalign")
Calls: calculate_tm_and_dg ... pairwiseAlignment -> .call_fun_in_pwalign -> .load_package_gracefully
Execution halted

You may need another install here. Can you also add this script to the run script?